test/jesse_tests_draft4_SUITE.erl

```diff
@@ -44,12 +44,20 @@ all() ->
 
 init_per_suite(Config) ->
   {ok, _} = application:ensure_all_started(jesse),
+  SkipList =
+    [
+      {<<"ref">>, <<"Location-independent identifier">>}
+    , {<<"ref">>, <<"Recursive references between schemas">>}
+    , {<<"ref">>, <<"Location-independent identifier with base URI"
+                    " change in subschema">>}
```

Comment:
so, 3 test-cases are skipped. Seems all 3 require "arbitrary document-local reference URI lookups" / "location indpendent identifiers", but it seems it would be very difficult to implement them efficiently. Naive way would be to just do recursive descent from `#state.root_schema`.